### PR TITLE
Task1

### PR DESCRIPTION
**Pull request by anjank81**

### CODE REVIEW

**Comment by anjank81** [comment]
Screenshot from 2019-10-23 12-00-16

**Comment by anjank81** [comment]
Screenshot from 2019-10-23 12-30-57

**Comment by anjank81** [comment]
Screenshot from 2019-10-23 12-57-46

**Comment by anjank81** [comment]
Screenshot from 2019-10-23 13-19-03

File: card1.html [hunk 1]
```diff
@@ -5,25 +5,44 @@
...
     </style>
   </head>
-  <body>
-    <div class="card">
-      <img src="as.jpg" alt="img" />
-      <div class="container">
-        <h4><b>Abcd</b></h4>
-        <p>Engineer</p>
+  <body bgcolor="grey">
+    <center style="padding-top:  150px;">
```
Comment (banupriya-SR-xelp) [inline card1.html]: add div and 
try to center the card using css

File: card3.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    </style>
+
+<body>
+
+
+
+<div class="card">
+    <img src="images/ru79eu.png" alt="alexa" class="avatar" id="img">
+    <div class="name">Alexndra Dadario</div>
+    <p class="title">Parker Industries</p>
```
Comment (banupriya-SR-xelp) [inline card3.html]: remove "p" tag everywhere 
use "div" and add  font size using css

**Review by banupriya-SR-xelp — CHANGES_REQUESTED** [review]

File: card3.html [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+
+
+<div class="card">
+    <img src="images/ru79eu.png" alt="alexa" class="avatar" id="img">
+    <div class="name">Alexndra Dadario</div>
+    <div class="title">Parker Industries</div>
+    <div class="round3">UX DESIGNER</div>
+    <div style="text-align:center;background-color: lightgray ; padding-top: 10px; color: grey" >
+        <h4>USUALLY COLLABORATED WITH</h4>
```
Comment (banupriya-SR-xelp) [inline card3.html]: remove h4 tag add css here

File: card3.html [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+        <img src="images/profile1.jpeg" class="dot"></img>
+        <img src="images/profile2.jpeg" class="dot"></img>
+        <img src="images/profile3.jpeg" class="dot"></img>
+        <img src="images/profile4.jpeg" class="dot"></img>
+        <sup class="dot"> +2</sup>
```
Comment (banupriya-SR-xelp) [inline card3.html]: instead of "sup" try to use css padding and margin property to align  the text

**Review by banupriya-SR-xelp — CHANGES_REQUESTED** [review]

**Review by banupriya-SR-xelp — APPROVED** [review]

**Pull request merged**